### ISSUE TITLE
Getting 404 for default device when trying to start or stop an app

### ISSUE THREAD

**Issue by TimPerry** [happyleavesaoc/python-firetv#16]
Running on raspberry pi, everything else seems to work like status endpoints but when trying to do an action or start/stop an app I get a 404. Which when looking through the source seems to be when a device is not recognised but this confuses me as I am using the default 

Example run command
```
firetv-server -d 192.168.1.4:5555
```

Example call:
```
http://192.168.1.18:5556/devices/default/apps/netflix/stop
```

Installed using pip:

```
pip show firetv
---
Name: firetv
Version: 1.0.2
Location: /usr/local/lib/python2.7/dist-packages
Requires: adb
```

**Comment by TimPerry** [comment]
Tried building on OS X via dockerfile you provide. When trying I now get this error:

```
('Received checksum %s != %s', (84025, 242512))
```

**Comment by happyleavesaoc** [comment]
@juliangeiges any thoughts on this 404?

@sytone any thoughts on that dockerfile error?

**Comment by TimPerry** [comment]
Would merging #11 fix this?

**Comment by arraylabs** [comment]
@TimPerry from what i can tell that Dockerfile is pulling the 1.02 version which i don't think has the start/stop items. Working on building a dockerfile for myself, and can publish here if i get it working.

**Comment by arraylabs** [comment]
@TimPerry I submitted a PR with my updated Dockerfile that uses the github master instead of the pip source for firetv-server.

**Comment by Dirt-Nasty** [comment]
@TimPerry @happyleavesaoc Is this still an issue? Im having a problem starting/stopping apps as well.  Installed using pip

**Comment by destructure00** [comment]
I'm having the same problem. Created a new issue here https://github.com/happyleavesaoc/python-firetv/issues/28 before finding this one.

**Comment by cbrherms** [comment]
Same issue here, except just locks up rather than 404 and no action performed

**Comment by cbrherms** [comment]
Should probably add installed via pip

**Comment by TimPerry** [comment]
Not sure if this is fixed but closing seen as #28 was closed.
